pom.xml

```diff
@@ -202,29 +200,6 @@
 			<type>test-jar</type>
 			<scope>test</scope>
 		</dependency>
-		<dependency>
```

Comment:
why all these dependencies are not needed anymore?

Reply:
i have tried execute tests with `3.0.0-M5` but there is some missing dependency which is needed.  In the next PR i will investigate what is the problem with that version but for now we can let this lower version (which is stable)

Reply:
These dependencies were needed for that `M5` version

Reply:
I have tried with 3.0.0-M5 but there is some dependency, which is missing. Anyway, in the next PR I can investigate the problem and change it to this 3.0.0-M5 but for now, we can let it on stable 2.22.2.